<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To-Do LIst</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
	background-color: #1F2A2F;
}

QScrollArea &gt; QWidget &gt; QWidget {
	background-color: #1F2A2F;
	border: none;
}

QScrollArea {
	border: none;
	margin: 0px;
	padding: 0px;
	background-color: #1F2A2F;
}

QScrollArea:vertical {
	border: none;
	background: #2E3F3A;
	width: 10px;
	margin: 0px 0px 0px 0px;
}

QScrollArea::handle:vertical {
	background: #A8D5BA;
	min-hieght: 20px;
	border-radius: 5px;
}

QScrollArea::add-line:vertical {
	border: none;
	background: #2E3F3A;
	height: 0px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollArea::sub-line:vertical {
	border: none;
	background: #2E3F3A;
	height: 0px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollArea::add-page:vertical, QScrollArea::sub-page:vertical {
	background: 2E3F3A;
}

QLineEdit, QTextEdit {
	background-color: #3F514B;
	color: #A8D5BA;
	border: none;
	border-radius: 12px;
	padding: 6px;
	font-size: 18px;
}

QPushButton {
	background-color: #4A7C59;
	color: white;
	border: none;
	border-radius: 12px;
	padding: 6px 12px;
}

QPushButton:hover {
	background-color: #5A9F73;
}

QListWidget {
	border: none;
	border-radius: 12px;
	background-color: #2E3F3A;
	color: #A8D5BA;
	padding: 6px;
}

QMenuBar {
    background-color: #1F2A2F;
    color: white;
}

QMenuBar::item {
    background-color: transparent;
    color: white;
}

QMenuBar::item:selected {
    background-color: #3F514B;
}

QMenu {
    background-color: #2E3F3A;
    color: white;
}

QMenu::item:selected {
    background-color: #3F514B;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="title_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>476</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Text Semibold</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="desc_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>581</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Text Semibold</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI Variable Text Semibold'; font-size:18px; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>601</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>491</width>
      <height>271</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="font">
         <font>
          <family>Segoe UI Variable Text Semibold</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="add_task_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Text Semibold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Task</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
